Docs: Clarify a bit more on SurrealQL running options in the getting started guide (#648)
surrealdb/docs.surrealdb.com: doc-surrealdb_versioned_docs/version-1.x/introduction/start.mdx

[SurrealQL](/docs/surrealdb/surrealql/) is the query language for SurrealDB which can be used to query data from your SurrealDB database. While this is not a requirement for getting started, it is helpful to familiarise yourself with some commands.

There are two main ways to use SurrealQL:

1. **Surrealist**: A graphical user interface (GUI) that can be accessed via a web browser or used as a desktop application.
2. **SurrealDB CLI**: A command-line interface that allows you to interact with SurrealQL directly from the terminal.

> For more information, you can visit [Surrealist](https://Surrealist.app), and the [SurrealDB CLI documentation](/docs/surrealdb/cli/).```

Reviewer: ```suggestion
For more information, you can visit [Surrealist](https://Surrealist.app), and the [SurrealDB CLI documentation](/docs/surrealdb/cli/).
```

[SurrealQL](/docs/surrealdb/surrealql/) is the query language for SurrealDB which can be used to query data from your SurrealDB database. While this is not a requirement for getting started, it is helpful to familiarise yourself with some commands.

> There are a few different ways to use SurrealQL. The [Surrealist app](https://Surrealist.app) is a GUI that can be used in a web browser or as a desktop app, and the [SurrealDB CLI](https://surrealdb.com/docs/surrealdb/cli/) can be used from the command-line.

Reviewer: I think a single paragraph might look a bit too wordy so broken it down a bit more. Let me know your thoughts @matthewjnield 
```suggestion
There are two main ways to use SurrealQL:

1. **Surrealist**: A graphical user interface (GUI) that can be accessed via a web browser or used as a desktop application.
2. **SurrealDB CLI**: A command-line interface that allows you to interact with SurrealQL directly from the terminal.

For more information, you can visit [Surrealist](https://Surrealist.app), and the [SurrealDB CLI documentation](/docs/surrealdb/cli/).```

</TabItem>
<TabItem value="CLI" label="Using CLI">

> Once you have your [database running](/docs/surrealdb/cli/start), you can use the [SurrealDB CLI](https://surrealdb.com/docs/surrealdb/cli/) to run your queries. To do so, run the following commands:

Reviewer: We use relative links to avoid a refresh. 
```suggestion
Once you have your [database running](/docs/surrealdb/cli/start), you can use the [SurrealDB CLI](/docs/surrealdb/cli/) to run your queries. To do so, run the following commands:
```
